src/plugins/cloudMetadata.nim

```diff
@@ -200,13 +200,18 @@ proc cloudMetadataGetrunTimeHostInfo*(self: Plugin, objs: seq[ChalkObj]):
   if isGoogleHost(vendor, resolv) and (
     isSubscribedKey("_GCP_INSTANCE_METADATA") or
     isSubscribedKey("_GCP_PROJECT_METADATA") or
+    isSubscribedKey("_OP_CLOUD_PROVIDER") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_IP") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_REGION") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_TAGS") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_ACCOUNT_INFO") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_SERVICE_TYPE") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_INSTANCE_TYPE")
   ):
+    isSubscribedKey("_OP_CLOUD_PROVIDER"):
+      # FIXME use enum
+      result["_OP_CLOUD_PROVIDER"] = pack("gcp")
```

Comment:
```suggestion
    # FIXME use enum
    result.setIfNeeded("_OP_CLOUD_PROVIDER", "gcp")
```

similar below

thats a more standard pattern across other plugins. this plugin specifically doesnt follow most of the conventions elsewhere